src/frontend/styles/Game.module.css

```diff
@@ -37,25 +37,65 @@
 
 .actionBar {
   margin-top: 1%;
-  border: 2px solid #0070f3;
+  /* border: 6px solid #0070f3; */
   min-height: 20vh;
   min-width: 70vw;
   display: flex;
   justify-content: space-around;
   align-items: center;
+  /* position: absolute; */
+  /* width: 1240px;
+height: 200.68px;
+left: 100px;
+top: 580.41px; */
```

Comment:
why do we have all of these commented out styles?

Reply:
they were from figma @aislinblack , i left them there as i was debugging but i can clean it up and remove